views/pages/apply/review.njk

```diff
@@ -1,41 +1,56 @@
 {% extends "layouts/main.njk" %}
 
-{% block title %}Review | {{ form.title }} | {% endblock %}
+{% block title %}{{ stepConfig.title }} | {{ form.title }} | {% endblock %}
 
 {% set bodyClass = 'has-static-header' %}
 
 {% block content %}
+    {% set pageAccent = 'pink' %}
     <main role="main" id="content">
-        <div class="content-box inner inner--wide-only accent--pink a--border-top">
+        <form action="" method="POST"
+            class="js-application-form js-application-form-review
+            content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top"
+            data-abandonment-message="{{ stepConfig.abandonmentMessage }}">
             <div class="u-constrained-content">
-                <h1 class="t1">{{ review.title }}</h1>
+                <h1 class="t1">{{ stepConfig.title }}</h1>
                 {% for step in summary %}
-                    <h2 class="t2 t--underline accent--pink">
+                    <h2 class="t2 t--underline accent--{{ pageAccent }}">
                         {{ step.name }}
-                        [<a href="{{ baseUrl }}/{{ loop.index }}">Edit</a>]
+                        [<a href="{{ baseUrl }}/{{ loop.index }}" class="js-application-form-review-edit">Edit</a>]
                     </h2>
                     {% for fieldset in step.fieldsets %}
-                        <dl class="review-list {% if fieldset.fields.length === 1 %}review-list--stacked{% endif %}">
-                            {% for field in fieldset.fields %}
-                                {% if field.value %}
-                                    <dt>{{ field.label }}</dt>
-                                    {% if field.type === 'textarea' %}
-                                        <dd class="s-prose" data-hj-suppress>{{ field.value | striptags(true) | escape | nl2br }}</dd>
-                                    {% elseif field.type === 'checkbox' %}
-                                        <dd data-hj-suppress>{{ field.value | joinIfArray(', ') }}</dd>
-                                    {% else %}
-                                        <dd data-hj-suppress>{{ field.value }}</dd>
-                                    {% endif %}
+                    <dl class="{{ classNames({
```

Comment:
This was me playing around with using https://www.npmjs.com/package/classnames in templates to clean up conditional class logic. It works, but is a bit of a tangent so will pull it out of this PR.